### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by ryanlovett** [berkeley-scf/stat999-quarto#2]
stat133 and stat151a have been added to siteimprove for accessibility monitoring. They have good scores, but a few things were flagged.

These are from stat133, but probably also affect stat151a.

### Color contrast does not meet enhanced/minimum requirement

- Flagged on all blue hyperlinks (3.97:1; needs 7:1),
- right-side menu (3.97:1)
- instructor’s name where text is blue and background is white.
- bCourses (pink), Ed (purple), R-Datahub (gray), blue callout
- This for the stat133/spring-2014 homepage which is 1 of 5 pages of occurrences: 
<img width="333" alt="Screenshot_2024-02-07_at_10 15 51_AM" src="https://github.com/berkeley-scf/stat999-quarto/assets/10248125/973c82ad-856d-4e50-9080-0e8273e7d50b">

### Link missing a text alternative

- Top-left site logo
- The git logo after STAT 133 beneath the top-left logo
- In the left-side menu, the “>” icon next to Units (?; I thought it was just a greater-than symbol)
